.github/workflows/build-pelorus.yml

```diff
@@ -131,3 +131,32 @@ jobs:
           registry: ${{ env.imageregistry }}/${{ env.imagenamespace }}
           username: ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_PASSWORD }}
+
+
+  build-webhook:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      # Setup S2i and Build container image
+      - name: Setup and Build
+        id: build-exporter
+        uses: redhat-actions/s2i-build@v2
+        with:
+          image: 'pelorus-webhook-exporter'
+          path_context: 'exporters'
```

Comment:
I think this ok, but I want to understand how this works. 

It builds the whole exporters package on it container? It build only webhook on it (and each exporter on the other ones)?

Reply:
Alright, checked it out, it builds everything. And then my question is: why have 4 equal images in quay? (the `pelorus-{{type}}-exporter` ones, basically what changes them is the `APP_FILE` environment variable)

I did the steps of the action, got `2si` executable from https://github.com/openshift/source-to-image and run `./s2i build exporters registry.access.redhat.com/ubi8/python-39 pelorus-webhook-exporter:latest --loglevel 2`. 

Then run `docker container run -ti --rm -e APP_FILE=webhook/app.py pelorus-webhook-exporter` to test out, and worked. Should we add a test job for checking the image entrypoint? (for the webhook looks easier, the other exporters need kube-config file, I think)

Reply:
Alright, we can think about it later. But I would like that we have a issue related to that, so we do not lose track.

Reply:
created it https://github.com/konveyor/pelorus/issues/862